src/ImageSharp.Web/DependencyInjection/ImageSharpBuilder.cs

```diff
@@ -14,7 +15,7 @@ internal class ImageSharpBuilder : IImageSharpBuilder
         /// Initializes a new instance of the <see cref="ImageSharpBuilder"/> class.
         /// </summary>
         /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
-        public ImageSharpBuilder(IServiceCollection services) => this.Services = services;
+        public ImageSharpBuilder(IServiceCollection services) => this.Services = services ?? throw new ArgumentNullException(nameof(services));
```

Comment:
If `IServiceCollection` is ever null you have bigger problems.

Reply:
Right, I'll remove this check again 😅